base_layer/core/src/transactions/transaction_components/output_features.rs

```diff
@@ -294,6 +294,25 @@ impl OutputFeatures {
         }
     }
 
+    pub fn for_contract_acceptance(
+        contract_id: FixedHash,
+        validator_node_public_key: PublicKey,
+        signature: Signature,
+    ) -> OutputFeatures {
+        Self {
+            flags: OutputFlags::CONTRACT_ACCEPT,
+            sidechain_features: Some(
+                SideChainFeaturesBuilder::new(contract_id)
```

Comment:
Suggest using the builder helper function on `SideChainFeatures` rather than importing the build struct directly
```suggestion
                SideChainFeatures::builder(contract_id)
```